go/interpreter/lfvm/instructions.go

```diff
@@ -917,10 +904,17 @@ func genericCreate(c *context, kind tosca.CallKind) error {
 		}
 	}
 
+	input, err := c.memory.getSlice(offset.Uint64(), sizeU64, c)
```

Comment:
Moving the `getSlice` after the balance check of lines 896-905 might introduce the following question: if create or create2 is called with insufficient balance, should the memory grow to fetch the initialization data and then skip the creation, or should it grow either way.

Before this change, memory grow either way. With this change, if there is not enough balance, memory does not grow.

Reply:
yeap, as you mentioned, CT (and hence geth) do expect the memory to grow even if there is not enough balance. So I will move get slice above the balance check.